### PR TITLE
Molly HW10

### PR DESCRIPTION
**Pull request by mollymahar**
(And class exercises done with Hadrien @hatsy)

### CODE REVIEW

File: numbered_lines.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+def num_lines():
+	with open('small.txt') as f:
+		with open('new.txt', 'w') as f2:
+			for index, line in enumerate(f):
+				f2.write(str(index + 1) + " " + line.strip() + "\n")
```
Comment (danielsgriffin) [inline numbered_lines.py]: Try `rstrip()`:
`  -- Football Coach`
`4 -- Football Coach`
notice the missing two leading spaces on the left

File: seventeen1.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+			result = "\nThere are no marbles left. You win!\n"
+	print result
+
+def human_turn(marbles):
+	"""This function takes one integer argument and returns another integer."""
+	while marbles >= 1:
+		human = raw_input("\nYour turn. How many marbles will you remove (1-3)? ")
+		try:
+			human = int(human)
+		except Exception:
```
Comment (danielsgriffin) [inline seventeen1.py]: Good. Though perhaps we should start trying `ValueError` here.

File: seventeen1.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+		computer = 1
+	print "Computer removed", computer, "marbles."
+	return computer
+
+
+# call main function
+def main():
+	seventeen()
+
+if __name__ == '__main__':
```
Comment (danielsgriffin) [inline seventeen1.py]: Good job.

File: seventeen2.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
+# imports
+
+# body
+def seventeen():
```
Comment (danielsgriffin) [inline seventeen2.py]: Perhaps this could usefully be more than one function.

File: seventeen2.py [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+		for idx, val in p1_plays.items():
+			p1_plays[idx] = [int(val) for val in p1_plays[idx]]
+	return p1_plays
+
+
+# call main function
+def main():
+	seventeen()
+
+if __name__ == '__main__':
```
Comment (danielsgriffin) [inline seventeen2.py]: Good.

**Comment by danielsgriffin** [comment]
Good job.

File: numbered_lines.py (hunk 1 above)
Comment (mollymahar) [inline numbered_lines.py]: I noticed those but didn't like them...(I could also keep them in by just not using strip() and not using the \n character.)

**Pull request closed without merging**